client/connections/ConnectionFactory.ts

```diff
@@ -1,33 +1,34 @@
 import Log from '@ulixee/commons/Logger';
 import IConnectionToCoreOptions from '../interfaces/IConnectionToCoreOptions';
 import ConnectionToCore from './ConnectionToCore';
-import RemoteConnectionToCore from './RemoteConnectionToCore';
+import RemoteServerConnectionToCore from './RemoteServerConnectionToCore';
```

Comment:
What do you think of calling this ConnectionToRemoteCoreServer? I REALLY liked our switch to ConnectionToClient and ConnectionToReplay in terms of having the name of what you connect to at the end.

Reply:
Done